platform-test/jvm/src/main/scala/util/Dat2Csv.scala

```diff
@@ -0,0 +1,60 @@
+package util
+
+import java.io.File
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.{ ActorMaterializer, IOResult }
+import akka.stream.scaladsl._
+import akka.util.ByteString
+import hmda.parser.fi.lar.LarDatParser
+import hmda.parser.fi.ts.TsDatParser
+
+import scala.concurrent.ExecutionContext
+
+/*
+Takes a .DAT file and converts to pipe delimited CSV (2017)
+ */
+object Dat2Csv {
```

Comment:
Is `Dat2Csv` just for convenience in making test files, or do you expect to use it in the app somewhere?